<launch>
<param name="side" value="1" />
<node name="shared_memory_interface" pkg="agree_esmacat_pkg" type="agree_ros_interface" output="screen" />
</launch>
